### DISCUSSION TITLE
Question about the installation

### DISCUSSION THREAD

**Discussion by dxh398** [erikkastelec/PDFScraper#13] [Q&A] [answered]
Hello,

I got the following error message when try to install it: 
"AttributeError: module 'configparser' has no attribute 'SafeConfigParser'. Did you mean: 'RawConfigParser'?"

I'm using Python v3.12. Can you please help? Thanks!

**Comment by dxh398** [comment]
Thanks for replying. I'm new to Python, and I'm not sure what the Home Assistant you mean. My laptop is running win10, I ran "pip install PDFScraper" then got the error during the installation. This is what ChatGPT told me:

The error you're encountering suggests that the package or module you're using is attempting to use the SafeConfigParser class from the configparser module, but that class has been deprecated and renamed to RawConfigParser in Python 3.

**Reply by erikkastelec** [reply]
Hi,
I initially thought that the issue relates to another repository. Sorry for the confusion. 
I am working on a solution. Issue is caused by incompatible versions of some libraries.

**Comment by dxh398** [comment]
No wonder I didn't see your previous comment here. Thanks so much for the help!

**Reply by erikkastelec** [reply, marked as answer]
I have released a new version to pypi that should work with Python version 3.12. If this is not the case let me know.

**Reply by dxh398** [reply]
It's fixed! Thank you so much!
